### PR TITLE
Create longest_common_subsequence.js

### PR DESCRIPTION
**Pull request by Harikrishnan6336**
**Fixes issue:**

**Changes:**

Added Longest common subsequence in JavaScript

### CODE REVIEW

File: code/dynamic_programming/src/longest_common_subsequence/longest_common_subsequence.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
+const Longest_Common_Subsequence = (arr1, arr2) => {
```
Comment (arnavb) [inline code/dynamic_programming/src/longest_common_subsequence/longest_common_subsequence.js]: follow camelCase for all variables/functions.

File: code/dynamic_programming/src/longest_common_subsequence/longest_common_subsequence.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+            when the last character is not same, take maximum obtained by adding 
+            one character to one of the subsequences.
+            */
+            else {
+                LCS_def[i][j] = Math.max(LCS_def[i - 1][j], LCS_def[i][j - 1])
+            }
+        }
+    }
+    
+    return `The length of longest subsequence is: ${LCS_def[arr1.length][arr2.length]}`;
```
Comment (arnavb) [inline code/dynamic_programming/src/longest_common_subsequence/longest_common_subsequence.js]: Return just the length itself. Do the interpolation when you run `console.log`.

File: code/dynamic_programming/src/longest_common_subsequence/longest_common_subsequence.js [hunk 3]
```diff
@@ -0,0 +1,41 @@
+const Longest_Common_Subsequence = (arr1, arr2) => {
+
+    let LCS_def = Array.from({ length: arr1.length + 1 },
+        () => Array.from({ length: arr2.length + 1 },
+            () => 0));
+    
+    for (let i = 0; i < arr1.length + 1; i++){
```
Comment (arnavb) [inline code/dynamic_programming/src/longest_common_subsequence/longest_common_subsequence.js]: Reformat your code using prettier.

**Comment by Harikrishnan6336** [comment]
@arnavb Plese review this

**Pull request closed without merging**